hw/xilinx/synth/tcl/mark_debug_nets.tcl

```diff
@@ -3,3 +3,6 @@
 
 # System master AXI interface
 set_property MARK_DEBUG 1 [get_nets -of [get_cells sys_master_u]]
+
+# RVM socket nets
+# set_property MARK_DEBUG 1 [get_nets -of [get_cells rvm_socket_u]]
```

Comment:
For the future, we may add a more user-friendly approach to nets debugging. Something similar to the xbar configuration flow.